src/provider/store/configureStore.dev.ts

```diff
@@ -0,0 +1,43 @@
+import {createStore, applyMiddleware, Store} from 'redux';
+import {composeWithDevTools} from 'remote-redux-devtools';
+import thunk from 'redux-thunk';
+import {persistStore, persistReducer, PersistConfig, Persistor} from 'redux-persist';
+
+import Storage from '../model/Storage';
+
+import {RootState} from './typings';
+import {ProviderMiddleware} from './middleware/ProviderMiddleware';
+import rootReducer from './root-reducer';
+
+
+const persistConfig: PersistConfig = {
+    key: 'root',
+    debug: true,
+    storage: Storage
+};
+
+const middleware = (injectables: {}) => applyMiddleware(
+    thunk.withExtraArgument<{}>(injectables),
+    ProviderMiddleware
+);
+
+const persistedReducer = persistReducer(persistConfig, rootReducer);
+
+const composeEnhancers = composeWithDevTools({realtime: true, port: 9950});
```

Comment:
When running this locally (without the redux tools enabled), I see `SocketProtocolError` and `sctransport.js:10 WebSocket connection to 'ws://localhost:9950/socketcluster/' failed` errors in the console. Does that happen for anyone else? Are those down to `remote-redux-devtools`?

Reply:
Yes, if you run `npm run start:redux` it will start the local server and you wont get this error. Its basically just trying to send the state to the devtool server so you can view it.